### PR TITLE
feat: support `--shell` argument

### PR DESCRIPTION
**Pull request by aslushnikov**
This lets us use `npx bisect-chrome --good XXX --bad YYY --shell "npm run ctest"

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -156,18 +164,22 @@ async function bisect(scriptPath, good, bad, isManual) {
   console.log(`RANGE: https://chromium.googlesource.com/chromium/src/+log/${fromSha}..${toSha}`);
 }
 
-function runScript(scriptPath, revisionInfo) {
+function runScript(shellCmd, scriptPath, env) {
   const log = debug('bisect:runscript');
   log('Running script');
-  const spawner = scriptPath.endsWith('.js') ? fork : spawn;
```
Comment (JoelEinbinder) [inline index.js:162]: this line should already be doing what you want with no need for `--shell`

File: index.js [hunk 2]
```diff
@@ -156,18 +164,22 @@ async function bisect(scriptPath, good, bad, isManual) {
...
-  const child = spawner(scriptPath, [], {
-    stdio: ['inherit', 'inherit', 'inherit', 'ipc'],
-    env: {
-      ...process.env,
-      CRPATH: revisionInfo.executablePath,
-      PUPPETEER_EXECUTABLE_PATH: revisionInfo.executablePath,
```
Comment (JoelEinbinder) [inline index.js:168]: and you deleted my wonderful environment variables :(

Comment (JoelEinbinder) [reply index.js:168]: oh I guess they are up above

**Review by JoelEinbinder — CHANGES_REQUESTED** [review]
I think the current code just works for you based on the file extension. Feel free to change the docs to clear up your confusion though.

File: README.md [hunk 3]
```diff
@@ -10,14 +10,18 @@ Parameters:
...
 - `<script>`  path to a Puppeteer script that returns a non-zero code for BAD and 0 for GOOD.
 
 Example:
 - `npx bisect-chrome --good 577361 --bad 599821 simple.js`
+- `npx bisect-chrome --good 577361 --bad 599821 --shell "npm run ctest"`
 - `npx bisect-chrome --manual --good 577361 --bad 599821`
 
 Use https://omahaproxy.appspot.com/ to find revisions.
 
+Note: the script exposes Chromium executable path as \`CRPATH\` environment variabble.
```
Comment (JoelEinbinder) [inline README.md:23]:
```suggestion
Note: The Chromium executable path is exposed to the script as the \`CRPATH\` environment variable.
```

**Review by JoelEinbinder — APPROVED** [review]

**Comment by mxschmitt** [comment]
Amazing feature!

**Pull request merged**